highfidelity/hifi-content
# Added baseball bat to pinata, updated cemetery candles, harp sound from spirit

## Test Plan (in hifi://Mexico)
1. Check that baseball bat is next to the pinata and collides with the pinata
2. Walk into the cemetery and see that candles light up when you move into their zones (including the entrance of the cemetery)
3. Wait for the spirit in the cemetery to grant you a marigold, you should hear a harp sound
